docs/Technical Reference.md

```diff
@@ -2,110 +2,102 @@ This page provides a technical reference on certain advanced configuration optio
 
-* **BucketTitle** - Human-friendly title for your bucket to be displayed in the dropdown menu.
+      Note that users with search access can see the entire contents of the bucket. Use with care.
 
-* **CertificateArn** - This is the arn for your AWS-managed SSL certificate for your QuiltWebHost domain. Use AWS Certificate Manager to set this up.
+8. Click Next.
+9. On the Options screen that follows, go to the "Termination Protection" section in "Advanced" and click "Enable".
 
-* **ConfigBucketName** - An unused bucket name for the template to create and populate with your navigator's configuration.
+    ![](./imgs/term_protect.png)
 
-* **QuiltBucketName** - The name of an existing S3 bucket you want to use with T4. It will store your T4 data.
+    This protects the stack deployment pipeline from accidental deletion. Click Next.
 
-* **QuiltWebHost** - The URL you want to use to access your navigator on the Internet. You must have a valid SSL certificate for this domain in `CertificateArn` if you want to use https.
+10. On the confirmation screen, check the box asking you to acknowledge that CloudFormation may create IAM roles, then click Create.
 
-* **Users** - A non-empty comma-separated list of IAM User ARNs that will grant permission to those users to search. A good first entry in this list is your own ARN, which you can find in IAM.
+    ![](./imgs/finish.png)
 
-* Notes - Updating BucketIcon, BucketDescription, or BucketTitle will not update your navigator configuration -- these parameters are only checked when a stack is being created at this time. You'll need to update your federation.json in your config bucket to update these values.
+    Click Create.
 
-1. Click Next
-1. On the Options screen, Under the Advanced section, next to Termination Protection, select the Enable check box. This protects the stack deployment pipeline from deletion. Then continue by clicking Next
+11. CloudFormation typically takes around 30 minutes to spin up your stack. Once that is done, you should see `CREATE_COMPLETE` as the Status for your CloudFormation stack.
 
-![](./imgs/term_protect.png)
+    ![](./imgs/outputs.png)
 
-1. Acknowledge that CloudFormation may create IAM roles
+12. Select the stack and open the Outputs tab. These should be two values there. One is labeled `CloudFrontDomain`. The other is labeled `LoadBalancerDNSName`. These are the endpoints for your catalog and your authorization services, respectively. They still need to be mapped to user-facing URLs via DNS.
 
-![](./imgs/finish.png)
+13. Go to your DNS service (if you are using AWS, this is [Route 53](https://aws.amazon.com/route53/)). Create two `CNAME` records: one mapping your catalog URL (`QuiltWebHost`) to the `CloudFrontDomain`, and one mapping your auth service URL (`RegistryHost`) to the `LoadBalancerDNSName`. Make sure that the value you set for `CloudFrontDomain` matches the value you set for `AllowedOrigin` in step 1.
```

Comment:
RegistryHost is no longer an input so we should add it as an output of the stack and explain where to find it in this section of the docs.

Reply:
OK. Has this not been added as an output of the stack yet?